alvr/server_core/src/tracking/mod.rs

```diff
@@ -471,6 +479,36 @@ pub fn tracking_loop(
             })
             .ok();
 
+        let publish_vmc = matches!(
+            SESSION_MANAGER.read().settings().headset.vmc,
+            Switch::Enabled(VMCConfig { publish: true, .. })
+        );
+        if publish_vmc {
+            let orientation_correction = matches!(
+                SESSION_MANAGER.read().settings().headset.vmc,
+                Switch::Enabled(VMCConfig {
+                    orientation_correction: true,
+                    ..
+                })
+            );
+
+            if let Some(sink) = &mut vmc_sink {
+                let tracking_manager_lock = ctx.tracking_manager.read();
+                let device_motions = device_motion_keys
+                    .iter()
+                    .filter_map(move |id| {
+                        Some((
+                            *id,
+                            tracking_manager_lock
+                                .get_device_motion(*id, timestamp)
```

Comment:
Don't really have the time to actually look right now, how does this interact with hand tracking/multi-input protocol (I forgot what it's called sry). I.e. does it miss the hand poses? And is this before and after the removal of the hand poses in the case of protocol compat?

Reply:
You can see the multimodal compat is at line 352. that already handles everything we need. after that point you don't care if the client does or doesn't support multimodal protocol, it will always be used.

Reply:
Great. Well then I think hands won't get exported properly if you're currently using hand tracking. Or I'm missing something

Reply:
This is only sending hand position/rotation information. Even if the protocol supports hand poses (through bone position/rotation), I haven't checked how to integrate that. So if there is any data about that, the sink should just ignore it for now.

Reply:
I'll look how it behaves with hand tracking when I get home and report back then.

Reply:
@The-personified-devil Ah i got it now, you're right

Reply:
Now I see what you mean! I started toying with hand tracking and integrating it wasn't hard, but noticed that they require their own rotation corrections :p

Reply:
That's odd. controllers do need a pose offset fix, which we do from settings. since you are getting the data from the tracking_manager then the pose is already fixed, and also recentered. same for hands. so i'm not sure a separate pose fox for hands

Reply:
in the end it was an off calibration of the Right Elbow what was off. It's kinda hard to calibrate without shoulder/upper arm data, but now it should be fine.